gradle.properties

```diff
@@ -2,12 +2,15 @@
 # because some versions here need to be matched by app authors in
 # their own projects. So don't get fancy with syntax!
 
+org.gradle.java.installations.auto-download=false
+org.gradle.jvmargs=-Dfile.encoding=UTF-8
+
 # Versioning
 cordaProductVersion = 5.0.0-INTEROP
 # NOTE: update this each time this module contains a breaking change
 ## NOTE: currently this is a top level revision, so all API versions will line up, but this could be moved to
 ##   a per module property in which case module versions can change independently.
-## IMPORTANT: 
+## IMPORTANT:
 ## The interop feature branches track api revisions separately to the mainline branch.
 ## API version of last merge from corda mainline: 731
 cordaApiRevision = 2
```

Comment:
Bump our API version to 3, also update the comment to contain the API version that is being merged.

Reply:
Do we really need it? we can use the 2 right?

Reply:
Nope. The purpose of the version number is to prevent people pulling incompatible dependencies. If there are breaking changes, it's got to be incremented!

Reply:
Following Alex comment, as in the meantime new commit was merge (https://github.com/corda/corda-api/commit/5c106353dcaac7c0c5969130a8611e655ea0efe2 updating `cordaApiRevision = 3`), so now it should be change to `cordaApiRevision = 4`

Reply:
Fixed